server/config/default.json

```diff
@@ -5,6 +5,5 @@
     "api": "http://localhost:3060",
     "app": "http://localhost:3000"
   },
-  "apiKey": "0ac43519edcf4421d80342403fb5985d",
-  "stripePublicKey": "pk_test_5olkhgG5FgJDHcGpJllmCj6z"
+  "apiKey": "0ac43519edcf4421d80342403fb5985d"
```

Comment:
That key is for communicating with the api. Going to merge this since we have already open sourced.